io/src/main/scala/sbt/io/IO.scala

```diff
@@ -46,7 +46,10 @@ object IO {
   val temporaryDirectory = new File(System.getProperty("java.io.tmpdir"))
 
   /** The size of the byte or char buffer used in various methods. */
-  private val BufferSize = 8192
+  private val BufferSize =
+    Option(System.getProperty("sbt.io.buffer.size")).map(_.toInt).getOrElse(8192)
```

Comment:
See https://github.com/sbt/sbt/blob/bcf5ded572711854b9ba28cec1e30114b428380b/main/src/main/scala/sbt/internal/SysProp.scala#L81-L92, which is a mini style guide for system properties. I'd suggest name `sbt.io.bufferbyte`?

In general, I wonder if we should copy-paste SysProp.scala to sbt.internal.io or something.

Reply:
thanks for paying attention!

I believe as is it does comply with style guidelines as articulated in SysProp.scala

 // System property style:
  //   1. use sbt. prefix
  //   2. prefer short nouns
  //   3. use dot for namespacing, and avoid making dot-separated English phrase
  //   4. make active/enable properties, instead of "sbt.disable."

and also existing values such as

``` 
  def taskTimings: Boolean = getOrFalse("sbt.task.timings")
  def taskTimingsOnShutdown: Boolean = getOrFalse("sbt.task.timings.on.shutdown")
  def taskTimingsThreshold: Long = long("sbt.task.timings.threshold", 0L)
  def taskTimingsOmitPaths: Boolean = getOrFalse("sbt.task.timings.omit.paths")
```

that being said, it's all cosmetics, it's up to you as maintainer to define final naming

Reply:
>  copy-paste SysProp.scala to sbt.internal.io 

looks a good suggestion, however this was far out of scope of my humble hack